packages/block-editor/src/components/block-pattern-setup/style.scss

```diff
@@ -84,7 +84,6 @@
 
 		.carousel-container {
 			overflow: hidden;
-			text-align: center;
```

Comment:
I didn't see any obvious negative impact of removing this rule. 🤷‍♂️

If is a reason it needs to be here, we override it for the `pattern-slide` class below instead.

Reply:
The only thing I can think is that this was here to centrally align the Spinner that appears while the patterns are loading. I don't know how important that really is though.

Reply:
In my testing, the spinner wasn't center aligned even with this rule. I'll push this change for now, and we can take a look at that separately. 👍